### PR TITLE
added badge for top sales products

### PR DESCRIPTION
**Pull request by Anton-88**
1. Added badge for top sales products (added flag for drawing the badge. by default it is false, until there is a condition by which it can be drawn - there will be another issue for this)
2. Moved the colors bar on a products from top-left corner to top-right 
3. Removed duplicated global variable

### CODE REVIEW

File: src/components/common/badge/productBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+function ProductBadge({ badgeContent }: Props) {
+    return (
+        <div className={st.productBadge}>
+            <CustomImage
+                className={st.svgWrapper}
+                src='/img/top-sales-badge/topSalesBadge.svg'
+                alt='TopBadge'
+                width={0}
+                height={0}
+            />
```
[the remark this answers is not in the document]
Comment (Anton-88) [reply src/components/common/badge/productBadge.tsx]: thx, I implemented your suggestions with next commit [b8cbc7f](https://github.com/boarlabsxyz/Kavoon/pull/60/commits/b8cbc7fa47be832416dff1ea4d172de47b9af19c)

File: src/components/common/badge/productBadge.module.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    position: absolute;
+    left: 12px;
+    top: 12px;
+    padding: 8px;
+    border-radius: 12px;
+    background-color: var(--white);
+  }
+
+.svgWrapper {
+    padding-right: 8px;
```
Comment (yuramax) [inline src/components/common/badge/productBadge.module.css:14]: "padding-right" is incorrect usage for this case, please see what result it outcomes:

top-sales-badge-review

Instead, you can use the `margin` property for this element or the `column-gap` property for the parent element.

**Review by yuramax — CHANGES_REQUESTED** [review]

File: src/components/common/badge/productBadge.module.css (hunk 2 above)
Comment (Anton-88) [reply src/components/common/badge/productBadge.module.css]: corrected the logo according to your suggestions [ff674c7] https://github.com/boarlabsxyz/Kavoon/pull/60/commits/ff674c7c355dde77665707bb7fb3591d38476224

File: src/icons/topSalesBadgeIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    />
+  </svg>
+);
```
[the remark this answers is not in the document]
Comment (Anton-88) [reply src/icons/topSalesBadgeIcon.tsx]: done this suggestion in the next commit [a42d1f4](https://github.com/boarlabsxyz/Kavoon/pull/60/commits/a42d1f487965acf388a9d3e404d85c49d4ca3669)

**Review by VitalyyP — APPROVED** [review]
LGTM

File: src/components/common/badge/productBadge.module.css (hunk 2 above)
Comment (yuramax) [reply src/components/common/badge/productBadge.module.css]: Looks good! 👍

**Pull request merged**